schema/artwork/index.js

```diff
@@ -407,6 +408,16 @@ export const artworkFields = () => {
         );
       },
     },
+    is_saved: {
+      type: GraphQLBoolean,
+      resolve: ({ id }, {}, request, { rootValue: {
+        accessToken, userID } }) => {
+        if (!accessToken) return false;
+        return savedArtworkLoader(userID, accessToken)
```

Comment:
Right so here we are new-ing up a `DataLoader`, and then calling the load. I suppose the options can be passed and parsed with as the key...